config/jenkins/configuration/jobs/oeedger8r-cpp/Ubuntu1604BuildClang-7.yml

```diff
@@ -0,0 +1,77 @@
+---
+jobs:
+  - script: |
+        folder("oeedger8r-cpp") {
+            displayName("oeedger8r-cpp")
+        }
+        pipelineJob("oeedger8r-cpp/Ubuntu1804BuildClang-7") {
+            description()
+            keepDependencies(false)
+            parameters {
+                stringParam {
+                    name("PULL_NUMBER")
+                    defaultValue("master")
+                    description("")
+                    trim(true)
+                }
+                stringParam {
+                    name("LINUX_VERSION")
+                    defaultValue("1804")
```

Comment:
Is this meant for 1804 or 1604?
Your file is 1604, but job name and param is 1804.

Reply:
Same with the other file

Reply:
Oh the joys of refactoring...

Thanks, changed!